src/caustics/lenses/multiplane.py

```diff
@@ -32,6 +32,16 @@ class Multiplane(ThickLens):
         List of thin lenses.
     """
 
+    _meta_params = {
+        **ThickLens._meta_params,
+        **{
+            "lenses": {
+                "default": [],
+                "description": "List of thin lenses.",
+            }
+        },
+    }
+
```

Comment:
```suggestion
```
'lenses' is not a meta params.. you wanna look out of calls with `self.add_params` only